### PR TITLE
Wrap confirmation and finality transactions into batch_all in Millau -> RialtoParachain bridge

### PR DESCRIPTION
**Pull request by svyatonik**
Unless I miss something, that is all we need for https://github.com/paritytech/parity-bridges-common/issues/1677#issuecomment-1438347928. ~Testing right now -> draft~ Yep - seems like that is enough

### CODE REVIEW

File: relays/messages/src/message_race_loop.rs [hunk 1]
```diff
@@ -573,17 +573,18 @@ pub async fn run<P: MessageRace, SC: SourceClient<P>, TC: TargetClient<P>>(
...
 					nonces_range,
 					P::target_name(),
+					if let Some(ref target_batch_transaction) = target_batch_transaction {
+						format!(
+							". This transaction is batched with sending the proof for header {:?}.",
+							target_batch_transaction.required_header_id(),
+						)
+					} else {
+						Default::default()
+					},
```
Comment (serban300) [inline relays/messages/src/message_race_loop.rs:586]: Nit: I would use `map` here:

```
					target_batch_transaction.as_ref().map(|tx| format!(
						". This transaction is batched with sending the proof for header {:?}.",
						tx.required_header_id())
					).unwrap_or_default()
```

**Review by acatangiu — APPROVED** [review]

**Review by serban300 — APPROVED** [review]

**Comment by serban300** [comment]
It seems that now the confirmation race is stalling:

image

**Comment by svyatonik** [comment]
@serban300 Do you use some customized version of relay/nodes? Or they're just our latest images from docker hub?

**Comment by serban300** [comment]
It should have been the latest version on master yesterday, compiled locally, but I might have done something wrong. I'll check again.

**Comment by svyatonik** [comment]
Ok, now we have it live on test deployments + I've started it locally. So if something is wrong there, hope we'll find it

**Comment by svyatonik** [comment]
@serban300 I also had ~3 minutes delay in confirmations delivery locally. Filed a separate issue #1901. Thanks for spotting that!

**Pull request merged**